lua/mason-registry/buf/init.lua

```diff
@@ -0,0 +1,35 @@
+local Pkg = require "mason-core.package"
+local platform = require "mason-core.platform"
+local _ = require "mason-core.functional"
+local github = require "mason-core.managers.github"
+local std = require "mason-core.managers.std"
+
+local coalesce, when = _.coalesce, _.when
+
+return Pkg.new {
+    name = "buf",
+    desc = [[A new way of working with Protocol Buffers]],
+    homepage = "https://buf.build",
+    languages = { Pkg.Lang.Proto },
```

Comment:
Wouldn't "Protobuf" be more accurate?

```suggestion
    languages = { Pkg.Lang.Protobuf },
```